flow-launcher.tera

```diff
@@ -0,0 +1,201 @@
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="pack://application:,,,/Themes/Base.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+    <Thickness x:Key="ResultMargin">0 0 0 8</Thickness>
+    <Style
+        x:Key="ItemGlyph"
+        BasedOn="{StaticResource BaseGlyphStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Foreground" Value="#{{ text.hex }}" />
+    </Style>
+    <Style
+        x:Key="QueryBoxStyle"
+        BasedOn="{StaticResource BaseQueryBoxStyle}"
+        TargetType="{x:Type TextBox}">
+        <Setter Property="FontSize" Value="24" />
+        <Setter Property="Foreground" Value="#{{ text.hex }}" />
+        <Setter Property="CaretBrush" Value="#{{ mauve.hex }}" />
+        <Setter Property="FontSize" Value="26" />
+        <Setter Property="Height" Value="42" /> 
+        <Setter Property="Padding" Value="0,0,0,0" />
+    </Style>
+    <Style
+        x:Key="QuerySuggestionBoxStyle"
+        BasedOn="{StaticResource BaseQuerySuggestionBoxStyle}"
+        TargetType="{x:Type TextBox}">
+        <Setter Property="Background" Value="#{{ mantle.hex }}" />
+        <Setter Property="Foreground" Value="#{{ surface1.hex }}" />
+        <Setter Property="FontSize" Value="26" />
+        <Setter Property="Height" Value="42" />
+        <Setter Property="Padding" Value="0,0,0,0" />
+    </Style>
+    <Style
+        x:Key="WindowBorderStyle"
+        BasedOn="{StaticResource BaseWindowBorderStyle}"
+        TargetType="{x:Type Border}">
+        <Setter Property="BorderThickness" Value="2" />
+        <Setter Property="BorderBrush" Value="#{{ if(cond=flavor.dark, t=surface0.hex, f=surface1.hex) }}" />
+        <Setter Property="CornerRadius" Value="12" />
+        <Setter Property="Background" Value="#{{ mantle.hex }}" />
+    </Style>
+    <Style
+        x:Key="WindowStyle"
+        BasedOn="{StaticResource BaseWindowStyle}"
+        TargetType="{x:Type Window}">
+        <Setter Property="Width" Value="576" />
+        <Setter Property="RenderOptions.ClearTypeHint" Value="Enabled" />
+    </Style>
+
+    <Style
+        x:Key="PendingLineStyle"
+        BasedOn="{StaticResource BasePendingLineStyle}"
+        TargetType="{x:Type Line}">
+        <Setter Property="Stroke" Value="#{{ green.hex }}" />
+    </Style>
+
+    <!--  Item Style  -->
+    <Style
+        x:Key="ItemTitleStyle"
+        BasedOn="{StaticResource BaseItemTitleStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Foreground" Value="#{{ if(cond=flavor.dark, t=subtext0.hex, f=text.hex) }}" />
+    </Style>
+    <Style
+        x:Key="ItemSubTitleStyle"
+        BasedOn="{StaticResource BaseItemSubTitleStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Foreground" Value="#{{ if(cond=flavor.dark, t=overlay0.hex, f=text.hex) }}" />
+        <Setter Property="FontSize" Value="13" />
+    </Style>
```

Comment:
Same as above. 

I think with some testing, you could come up with a single colour that looks good on all flavours, resulting in the removal of the if condition.